auth/src/main/java/org/aerogear/mobile/auth/configuration/KeycloakConfiguration.java

```diff
@@ -90,6 +94,16 @@ public String getHostUrl() {
         return this.serverUrl;
     }
 
+    /**
+     * Get the Auth Server URL string of the Keycloak singleThreadService This URL should only be
+     * used to get the issuer field for token verification.
+     *
+     * @return the Auth Server URL of the Keycloak singleThreadService
+     */
+    public String getAuthHostURL() {
```

Comment:
If this is only used for issuer verification, would it better to rename it to `getIssuerHostname` to avoid confusion?